src/application.js

```diff
@@ -198,6 +225,15 @@ class XtkSchemaNode {
          * @type {string}
          */
         this.nodePath = this._getNodePath(true)._path;
+        /**
+         * Element of type join have a table of join
+         * @type {@type {XtkJoin[]}}
+         */
+        this.join = [];
```

Comment:
This should be named "joins" as there is more than one join

Reply:
done thx